### PR TITLE
README.md Spacing Fixes

### PR DESCRIPTION
**Pull request by baweaver**
Noticed there was some odd indentation in the README, quick patch to normalize that.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,22 +50,24 @@ Supported Ruby Interpreters
...
 ### or to install the latest development version from github master
 
-    git clone http://github.com/bblimke/webmock.git
-    cd webmock
-    rake install
+```bash
+git clone http://github.com/bblimke/webmock.git
+cd webmock
+rake install
+```
```
Comment (baweaver) [inline README.md:70]: Granted this one is more of replacing the alternate 4-space indent codeblock style, but seeing as the rest of the doc was using backticks this seemed to make sense.

File: README.md [hunk 2]
```diff
@@ -383,7 +385,8 @@ RestClient.post('www.example.net', 'abc')    # ===> "abc\n"
 
 ### Dynamically evaluated raw responses recorded with `curl -is`
 
-    `curl -is www.example.com > /tmp/www.example.com.txt`
+`curl -is www.example.com > /tmp/www.example.com.txt`
```
Comment (baweaver) [inline README.md:388]: This causes the `curl` request to show up in a code block with backticks around it.

**Comment by baweaver** [comment]
Apparently past me had a similar idea: https://github.com/bblimke/webmock/pull/1021

...though that missed the curl issue.

**Comment by bblimke** [comment]
Thank you @baweaver :)

**Pull request merged**